src/scheduler/fifo.c

```diff
@@ -2379,8 +2379,6 @@ sched_settings_frm_svr(struct batch_status *status)
 	char *tmp_comment = NULL;
```

Comment:
Add a @MT-Safe: No in the block comment.  You use static variables.

Reply:
Done